index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Anja Schmidt's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="mobile_container">
+      <!-- Nav Bar -->
+      <nav>
+        <ul>
+          <li>Welcome</li>
+          <li><img class="menu" src="images/menu.png" alt="Menu" /></li>
+        </ul>
+      </nav>
+
+      <!-- Header -->
+      <header>
+        <section class="header_container">
+          <h1 class="heading1">Hey there. I’m Anja.</h1>
+          <h1 class="heading2">I’m a software developer</h1>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="social_container">
+            <a
+              href="https://twitter.com/Anja_Schmidt7"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><img src="images/twitter.svg" alt="Twitter"
+            /></a>
+            <a
+              href="https://twitter.com/Anja_Schmidt7"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><img src="images/github.svg" alt="Github"
+            /></a>
+            <a
+              href="https://twitter.com/Anja_Schmidt7"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><img src="images/Linkedin-icon.svg" alt="linkedin"
+            /></a>
+          </div>
```

Comment:
- [ ] Please kindly put this lists of icons in a `<ul>` tag to group them, `<li>` tag per icon, each icon should have a link (`<a>` tag). Kindly fix that,  as this makes your html more semantic.